libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -48,7 +47,7 @@ class AC_PosControl
 public:
 
     /// Constructor
-    AC_PosControl(const AP_AHRS_View& ahrs, const AP_InertialNav& inav,
+    AC_PosControl(AP_AHRS_NavEKF& ahrs, const AP_AHRS_View& ahrs_view,
```

Comment:
@khancyr I think you didn't understood my answer. We need ahrs_view, but I think we don't need ahrs - we just need to confirm that with @tridge.